gems/aws-sdk-s3/spec/plugins/streaming_retry_spec.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+module Aws
+  module S3
+    module Plugins
+      describe StreamingRetry do
+        let(:creds) { Aws::Credentials.new('akid', 'secert') }
```

Comment:
Super important spelling error

Reply:
The funny thing is that I copied this from another spec.  I've gone and updated this across the package - it seems to have made it into a few places...